src/xrGame/MainMenu.h

```diff
@@ -72,6 +72,8 @@ class CMainMenu : public IMainMenu,
 
     xr_vector<CUIWindow*> m_pp_draw_wnds;
 
+    bool languageChanged;
```

Comment:
We can change it, but while it isn't changed we should follow the current style.
Visual Studio (or ReSharper?) has a different colours for local, global and member variables. If there's no colours separation – come on, looking for declaration is less than minute!

The only thing I'm strongly against of is mixing `camelCase` and `lower_case`. (yes, i believe that this is a mixing)
Variables should be named like `m_language_changed` either `mLanguageChanged` but not `m_languageChanged`!

Changing the code style should be discussed with all team members. @OpenXRay/xray-contributors

Reply:
I think, we should merge this PR with the current code style and create a separate issue so that this PR does not become a burden. It is related to ingame language change feature, not the code style.

Reply:
The issue is created: #278.
Code style discussion should be continued there. This PR will be merged with the current style.